Clean-up edits for the Introduction page (#191)
pinot-contrib/pinot-docs: README.md

### User-Facing Real-Time Analytics

> **User-facing analytics**, or site-facing analytics, refers to the analytical tools and applications exposed directly to the end users of your product. In a user-facing analytics application, the user base is _all_ end users of an app – not just a few analysts doing offline analysis or a handful of data scientists in a company running ad hoc queries. Each end user receives personalized analytics on their personal devices (think hundreds of thousands of queries per second). Because these queries are triggered by apps, and not written by people, the scale grows in proportion to the number of active users on the app (think millions of events per second).

Reviewer: ```suggestion
**User-facing analytics**, or site-facing analytics, refers to the analytical tools and applications exposed directly to the end users of your product. In a user-facing analytics application, all users receive personalized analytics on their devices (think hundreds of thousands of queries per second). Because these queries are triggered by apps, and not written by people, the scale grows in proportion to the number of active users on the app (millions of events per second).
```

> This is designed for when you need the freshest possible data, which leads into the second aspect – **real-time analytics**. For some use cases, data from yesterday is no longer useful. The data needs to be available for analytics as soon as it is generated (think latencies under one second).

Reviewer: ```suggestion
Pinot is designed to provide the freshest possible data and supports **real-time analytics**. Data generated in Pinot is immediately available for analytics in latencies under one second.
```

### Why is user-facing real-time analytics is so challenging?

> What does the user-facing real-time analytics app described above require from the underlying infrastructure?

Reviewer: ```suggestion
What does user-facing real-time analytics require from the underlying infrastructure?
```

Reviewer: this intro question should align with the heading---which is a better fit for the content it's describing?

### Companies using Pinot

> LinkedIn, where Pinot originated, currently has one of the largest Pinot deployments, powering more than 50+ user-facing applications (such as Viewed My Profile, Talent Analytics, Company Analytics, Ad Analytics and many more). At LinkedIn, Pinot also serves as the backend to visualize and monitor 10,000+ business metrics.

Reviewer: typically, this content is outside of docs scope and belongs on the marketing/main site

Author: I totally agree. Baby stepping in the right direction, but ultimately we need to remove it.

**User-facing analytics products**

> Pinot is the perfect choice for user-facing analytics products. Pinot was originally built at LinkedIn to power rich interactive real-time analytics applications, such as [Who Viewed Profile](https://www.linkedin.com/me/profile-views/urn:li:wvmp:summary/), [Company Analytics](https://www.linkedin.com/company/linkedin/insights/), [Talent Insights](https://business.linkedin.com/talent-solutions/talent-insights), and many more. [UberEats Restaurant Manager](https://eng.uber.com/restaurant-manager/) is another example of a customer-facing analytics app built with Pinot.

Reviewer: pieces of this repeated in line 64, we could consolidate

**Real-time dashboard for business metrics**

> Pinot can be also be used to perform typical analytical operations such as slice and dice, drill down, roll up, and pivot on large scale multi-dimensional data. For instance, at LinkedIn, Pinot powers dashboards for thousands of business metrics. You can also connect various BI tools such as Superset, Tableau, or PowerBI to visualize data in Pinot.

Reviewer: good opportunity to add links to definitions in glossary--and links to BI tools

> Instructions to connect Pinot with Superset can be found [here](https://docs.pinot.apache.org/integrations/superset).

Reviewer: ```suggestion
To connect Pinot with Superset, see [Superset docs](https://docs.pinot.apache.org/integrations/superset).
```

Reviewer: Global edit--rename integration pages 
Superset >> Integrate Pinot and Superset
